lib/safe_pusher/cli.rb

```diff
@@ -1,145 +1,73 @@
 module SafePusher
   class CLI
-    SHORTCUTS = {
-      't' => 'test',
-      'l' => 'lint',
-      'p' => 'push',
-      'o' => 'open',
-      'm' => 'amend',
-      'a' => 'add',
-      'c' => 'commit',
-    }.freeze
-
-    private_constant :SHORTCUTS
-
     def initialize(arguments:)
       @arguments = arguments
     end
 
     def start
       return version if arguments.first == '--version'
 
-      unless arguments_valid?
-        help
-
-        return
-      end
+      help if commands.include?(nil)
 
-      arguments.each do |command|
-        execute_command(command)
-      end
+      commands.compact.each { |command| execute_command(command) }
     end
 
     private
 
     attr_reader :arguments
 
     def execute_command(command)
-      if SHORTCUTS[command]
-        send(SHORTCUTS[command])
-      else
-        send(command)
-      end
-    end
-
-    def test
-      puts '##########################'.yellow
-      puts '## Testing new files... ##'.yellow
-      puts '##########################'.yellow
-      results = SafePusher::RspecRunner.new.call
-
-      exit results unless results == 0
-    end
-
-    def lint
-      puts '#######################'.yellow
-      puts '## Running linter... ##'.yellow
-      puts '#######################'.yellow
+      explain(command) if verbose
 
-      results = SafePusher::ProntoRunner.new.call
+      results = Kernel
+        .const_get("SafePusher::Client::#{services[command].capitalize}")
```

Comment:
We can remove the interpolation by turning this into `SafePusher::Client.const_get(services[command].capitalize)`